lanelet2/lanelet2_core/include/lanelet2_core/primitives/RegulatoryElement.h

```diff
@@ -166,6 +166,21 @@ class MutableParameterVisitor : public boost::static_visitor<void> {  // NOLINT
   virtual ~MutableParameterVisitor() = default;
   std::string role;  //!< applyVisitor will set the current role here
 };
+/**
+ * @brief You can inherit from this visitor to perform an operation on and alter each
+ * parameter of a regulatory element 
+ * @see RegulatoryElement::applyVisitor
+ */
+class TrueMutableParameterVisitor : public boost::static_visitor<void> {  // NOLINT
```

Comment:
Now there is a MutableParameterVisitor and a TrueMutableParameterVisitor. What makes this one "True" and the other "not true"? Is there a clearer name which could be used?

Reply:
Yeah I realized that this existing function's name could either mean a function that visits "mutable parameter" (which is what it was), or a "mutable function" that visits parameter. I have renamed my visitor to imply more that it can edit the parameters.